packages-exp/auth-types-exp/index.d.ts

```diff
@@ -1080,6 +1277,14 @@ export interface User extends UserInfo {
    * @param forceRefresh - Force refresh regardless of token expiration.
    */
   getIdToken(forceRefresh?: boolean): Promise<string>;
+  /**
+   * Returns a de-serialized JSON Web Token (JWT) used to identitfy the user to a Firebase service.
+   *
+   * Returns the current token if it has not expired. Otherwise, this will refresh the token and
```

Comment:
Perhaps "Returns the current token if it has not expired or if it will not expire in the next five minutes."